public/js/index.js

```diff
@@ -1,5 +1,32 @@
-function displayMessage() {
-  return 'Hello, World!';
+function buildProduct(product) {
+  var div = document.createElement("div");
```

Comment:
I don't sure if that would be a good idea... I prefer to treat the "product" as an atom/unit because for instance  `createProductDiv` reveals implementation details.